authorization-service/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "backend_shop",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "deploy": "sls deploy",
+    "remove": "sls remove",
+    "local": "sls offline start",
+    "test": "cross-env REGION=eu-west-1 jest import-service/src/handlers/importProductsFile/handler.test.ts",
```

Comment:
Lest use generic script, not pointed to single file, especially in import service

Reply:
done